src/allocator/fam_allocator_client.cpp

```diff
@@ -316,29 +316,51 @@ void Fam_Allocator_Client::wait_for_copy(void *waitObj) {
     return famCIS->wait_for_copy(waitObj);
 }
 
-void *Fam_Allocator_Client::backup(Fam_Descriptor *src, char *BackupName) {
+void *Fam_Allocator_Client::backup(Fam_Descriptor *src, const char *BackupName) {
     Fam_Global_Descriptor globalDescriptor = src->get_global_descriptor();
     uint64_t srcRegionId = globalDescriptor.regionId & REGIONID_MASK;
     uint64_t srcMemoryServerId = src->get_memserver_id();
     uint64_t srcOffset = globalDescriptor.offset;
-    return famCIS->backup(srcRegionId, srcOffset, srcMemoryServerId, BackupName,
+    
+    void *waitObj;
+    try {
```

Comment:
What is the reason for catching this exception ? Why not just pass the exception to above modules ?